### PR TITLE
Critical update to the Rari Capital space

### PR DESCRIPTION
**Pull request by davidlucid**
Critical update to the Rari Capital space to enable voting with unclaimed private RGT allocations.

### CODE REVIEW

File: spaces/rari/index.json [hunk 1]
```diff
@@ -41,6 +41,35 @@
           "type": "function"
         }
       }
+    },
+    {
+      "name": "contract-call",
+      "params": {
+        "address": "0xA54B473028f4ba881F1eD6B670af4103e8F9B98a",
+        "symbol": "RGT",
```
Comment (bonustrack) [inline spaces/rari/index.json:49]: Can you use another symbol? This must be unique.

Comment (davidlucid) [reply spaces/rari/index.json]: Ideally, they use the same symbol, but for now, I have changed the symbol of the vested RGT to vRGT.

**Review by gokaiorg — COMMENTED** [review]
If you change the strategies your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

**Comment by davidlucid** [comment]
Yes, if I cannot use the same symbol for the two `contract-call` strategies, then yes, I am fine with this as is.

**Review by gokaiorg — APPROVED** [review]

**Comment by davidlucid** [comment]
Is there any way to use the same symbol for the two `contract-call` strategies, or should I make a custom strategy, or should I make a single contract call that combines the two calls? @gokaiorg

**Pull request merged**